options/options.c

```diff
@@ -213,6 +222,8 @@ const struct m_sub_options vo_sub_opts = {
         .x11_bypass_compositor = 2,
         .mmcss_profile = "Playback",
         .fullscreen = HAVE_RPI ? 1 : 0,
+        .drm_connector_spec = NULL,
+        .drm_mode_id = 0,
```

Comment:
I'll remove it, because almost no other field is explicitly 0 initialized and that reduces code complexity